QuickSort.c

```diff
@@ -29,23 +37,22 @@ void quickSort(int arr[], int first, int last)
 
     }
 }
+void displayData(int arr[], int n)
+{
+    for (int i = 0; i < n; i++)
+    {
+        printf("%d", arr[i]);
+    }
+}
 int main()
 {
     int n, i;
     printf("Enter the lenght of array: ");
     scanf("%d", &n);
     int arr[n];

```

Comment:
I was expecting to see all of the above in a separate function (`readData`). I'm not sure how it all works in C, but ideally, I'd only like to see three things:
* read data
* perform sorting
* display data
Nothing else, no scans, no printfs, etc... See if you can achieve this. If not, it's OK...

Reply:
Actually there is no way to get variable from function to main without first initializing variable in main, maybe only with global variable but that would be a diffrent story.